From 0a7791e5dda5a096c611cf9d045db104fdcb7ae8 Mon Sep 17 00:00:00 2001
From: Tai Nguyen <ttnguyen@apm.com>
Date: Fri, 15 Jul 2016 10:38:02 -0700
Subject: [PATCH 25/59] MAINTAINERS: Add entry for APM X-Gene SoC PMU driver

Git-commit: fd3a628e3f2a04acb85b9169df1811627ba367f8
Patch-mainline: v4.9-rc1
References: fate#319482

This patch adds the MAINTAINERS entry for APM X-Gene SoC PMU driver.

Signed-off-by: Tai Nguyen <ttnguyen@apm.com>
Signed-off-by: Mian Yousaf Kaukab <yousaf.kaukab@suse.com>
---
 MAINTAINERS | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index c8f8abac7a6a..1984a7d29990 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -812,6 +812,13 @@ F:	drivers/net/phy/mdio-xgene.c
 F:	Documentation/devicetree/bindings/net/apm-xgene-enet.txt
 F:	Documentation/devicetree/bindings/net/apm-xgene-mdio.txt
 
+APPLIED MICRO (APM) X-GENE SOC PMU
+M:	Tai Nguyen <ttnguyen@apm.com>
+S:	Supported
+F:	drivers/perf/xgene_pmu.c
+F:	Documentation/perf/xgene-pmu.txt
+F:	Documentation/devicetree/bindings/perf/apm-xgene-pmu.txt
+
 APTINA CAMERA SENSOR PLL
 M:	Laurent Pinchart <Laurent.pinchart@ideasonboard.com>
 L:	linux-media@vger.kernel.org
-- 
2.11.0

